modules/aws-outposts-load-balancer-clb-fix-subnet.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb-fix-subnet_{context}"]
+= Ensuring AWS load balancers use the correct subnet
+
+When you extend an AWS VPC cluster to an Outpost, the subnet discovery by the cloud controller manager might cause new AWS Classic Load Balancers and Network Load Balancers to be added to the Outpost availability zone.
+Because AWS Outposts does not support these types of load balancers, you must prevent them from being added to the Outpost subnet by adding an annotation with the cloud-based subnets.
+
+[NOTE]
+====
+Manually ensuring that new Classic Load Balancers and Network Load Balancers use the correct subnets is a workaround for clusters installed with {product-title} {product-version}.
+====
+
+.Prerequisites
+
+* You have installed an {product-title} cluster into a custom VPC on AWS.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have created a user workload in the Outpost.
+
+.Procedure
+
+. Configure the load balancer `service` resource:
++
+.Example service configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: <application_name>
+  name: <application_name>
+  namespace: <application_namespace>
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-subnets: <aws_subnet> # <1>
+    service.beta.kubernetes.io/aws-load-balancer-internal: "true" # <2>
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: <application_name>
+  type: LoadBalancer
+----
+<1> Specify the cloud-based subnet.
+<2> Specify this annotation if the cloud-based subnet is private.
```

Comment:
Odds on most people copying and pasting and ignoring this caveat? Do we expect more services to be public or private?

Reply:
I can show two examples with the public/private as part of the heading to make it require less attentive reading :)